### ISSUE TITLE
段错误 (核心已转储)

### ISSUE THREAD

**Issue by SunNoJJ** [JaidedAI/EasyOCR#1305]
ted since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/sdb2/Software/andconda3Install/envs/molannotator/lib/python3.11/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=ResNet50_Weights.IMAGENET1K_V1`. You can also use `weights=ResNet50_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
/sdb2/Software/andconda3Install/envs/molannotator/lib/python3.11/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at /opt/conda/conda-bld/pytorch_1682343970094/work/aten/src/ATen/native/TensorShape.cpp:3483.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
/sdb2/Software/andconda3Install/envs/molannotator/lib/python3.11/site-packages/torch/utils/checkpoint.py:31: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")
段错误 (核心已转储)

**Comment by SunNoJJ** [comment]
需要将：
/sdb2/Software/andconda3Install/envs/molannotator/lib/python3.11/site-packages/easyocr/craft_utils.py
31行connectivity=4改为connectivity=8
nLabels, labels, stats, centroids = cv2.connectedComponentsWithStats(text_score_comb.astype(np.uint8), connectivity=8)
